Solutions/MimecastSEG/Analytic Rules/MimecastSIEM_AV.yaml

```diff
@@ -0,0 +1,54 @@
+id: 0f0dc725-29dc-48c3-bf10-bd2f34fd1cbb
+name: Mimecast Secure Email Gateway - AV
+description: Mimecast Secure Email Gateway - AV
+severity: Informational
+requiredDataConnectors:
+  - connectorId: MimecastSIEMAPI
+    dataTypes:
+      - MimecastSIEM_CL
+enabled: true
+query: MimecastSIEM_CL| where mimecastEventId_s=="mail_av"
+queryFrequency: 5m
+queryPeriod: 15m
+triggerOperator: gt
+triggerThreshold: 0
+suppressionDuration: 5h
+suppressionEnabled: false
+tactics:
+- Execution

```

Comment:
Please add `relevantTechniques`, Relevant [MITRE Techniques](https://attack.mitre.org/versions/v11/matrices/enterprise/) ID. 

This needs to be added in all the analytical rules of the PR. 

Example :- https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Azure%20Active%20Directory/Analytic%20Rules/FailedLogonToAzurePortal.yaml